README .R

```diff
@@ -0,0 +1,28 @@
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+source("1_fetch/src/fetch_data.R")
+source("2_process/src/process_data.R") 
+source("3_visualize/src/visualize_data.R")
+
+project_output_dir <- '1_fetch/out'
+mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+
+fetch_data(mendota_file)
+
+
+
+project_output_dir <- '2_process/out'
```

Comment:
I would like to see this value of `project_output_dir` formally passed (as an argument) to `summary_model()`, which is the one place where I believe this value gets used.

Reply:
see also the comment at the definition of `summary_model`